### PR TITLE
Added check for files exists

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eks-distro-base/update_base_image_other_repos.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ EXTRA_PR_BODY=""
 
 REPOS=(eks-distro eks-anywhere-build-tooling eks-anywhere)
 for repo in "${REPOS[@]}"; do
-    ${SCRIPT_ROOT}/../pr-scripts/update_local_branch.sh "$repo"    
+    ${SCRIPT_ROOT}/../pr-scripts/update_local_branch.sh "$repo"
```
Comment (kschumy) [inline eks-distro-base/update_base_image_other_repos.sh:34]: Intellij apparently made this change

File: eks-distro-base/update_base_image_other_repos.sh [hunk 2]
```diff
@@ -55,7 +55,12 @@ do
             ${SCRIPT_ROOT}/../pr-scripts/update_image_tag.sh "$repo" '.*' $IMAGE_TAG $BASE_IMAGE_TAG_FILE
 
             if [ "$(git -C ${OTHER_CLONE_ROOT}/${ORIGIN_ORG}/${repo} status --porcelain -- $BASE_IMAGE_TAG_FILE | wc -l)" -gt 0 ]; then
-                UPDATE_PACKAGES="$(cat ${SCRIPT_ROOT}/../eks-distro-base-updates/${key#al}/update_packages-${image})"
+                update_packages_path="${SCRIPT_ROOT}/../eks-distro-base-updates/${key#al}/update_packages-${image}"
+                if [ ! -f "$update_packages_path" ]; then
+                  echo "File for update packages does not exist: ${update_packages_path}"
+                  exit 1
```
Comment (jaxesn) [inline eks-distro-base/update_base_image_other_repos.sh:61]: I probably wouldnt fail, i would just skip.

Comment (kschumy) [reply eks-distro-base/update_base_image_other_repos.sh]: done

**Pull request merged**